crates/solidity/testing/snapshots/cst_output/AssemblyStatement/evm_builtins/input.sol

```diff
@@ -0,0 +1,152 @@
+assembly {
```

Comment:
That test always fails (in all versions), since there is never a version where all of them are legal.

I suggest splitting them into one test per keyword, or maybe two (using as a variable name, and in a function call), so that the test outputs are meaningful. Otherwise, I worry that changes to the 20k+ lines of snapshots will always be ignored during review, as it is really hard to understand visually.